output.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include<iostream>
+#include<fstream>
+#include<vector>
+#include<iomanip>
+using namespace std;
+
+void outputTextFile(vector <pair<string, string>> x)
+{
+  ofstream outputFile;
+  outputFile.open("output.txt");
+  outputFile << "       Output:       " << endl;
+  outputFile << "Token          Lexeme" << endl;
+  outputFile << "---------------------" << endl;
```

Comment:
This is actually my bad but, can you remove this? 
I know I said that to put it there but I forgot to look at his actual sample output.